packages/zendesk-client/src/use-zendesk-messaging-bindings.ts

```diff
@@ -0,0 +1,60 @@
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import type { HelpCenterSelect } from '@automattic/data-stores';
+
+export function useZendeskMessagingBindings(
+	HelpCenterStore: string,
```

Comment:
nitpick buy maybe this would be better
```suggestion
	HELP_CENTER_STORE: string,
```

Reply:
This is not a constant anymore so I don't think it should be caps.